server/athenian/api/models/web/calculated_pull_request_metrics_item.py

```diff
@@ -37,7 +37,7 @@ def for_(self):
         """Gets the for_ of this CalculatedPullRequestMetricsItem.
 
         :return: The for of this CalculatedPullRequestMetricsItem.
-        :rtype: ForSet
+        :rtype: ForSetPullRequests
```

Comment:
Actually, this should go to a proper type hint in the function signature. Was left from 2 years ago.